admin/src/client.rs

```diff
@@ -0,0 +1,28 @@
+use anyhow::{Context, Result};
+
+pub struct Client {
+    api_url: String,
+    api_key: String,
+}
+
+impl Client {
+    pub fn new(api_url: String, api_key: String) -> Self {
+        Self { api_url, api_key }
+    }
+
+    pub async fn revive(&self) -> Result<String> {
+        self.get("/admin/revive").await
```

Comment:
Might want to watch for #460 if @oddgrd changes the endpoint verb before merging this.